keps/sig-network/563-dual-stack/README.md

```diff
@@ -250,7 +250,10 @@ Phase 3 (Planned Kubernetes 1.17)
 - Dual-stack support for kubeadm
 - Expand container runtime support (containerd, CRI-O)
 
-Phase 4 and beyond
+Phase 4 (Planned Kubernetes 1.22)
+- Downward API support for Pod.Status.HostIPs
```

Comment:
This needs more design spec - doesn't it?

Reply:
It has been completed. There are two main points. Follow the PodIPs steps to add the logic of HostIPs and the downward API support for status.hostIP. 

- Types conversion https://github.com/kubernetes/kubernetes/pull/101566/commits/c8a5513a0b6cd3ef5af802c44b2878faf33b1508
- Downward API support for status.hostIPs https://github.com/kubernetes/kubernetes/pull/101566/commits/3281e7e72b39119de54773cb679d096e7b34fd1b

Reply:
It has been completed. There are two main points. Follow the PodIPs steps to add the logic of HostIPs and the downward API support for status.hostIP. 

- Types conversion https://github.com/kubernetes/kubernetes/pull/101566/commits/c8a5513a0b6cd3ef5af802c44b2878faf33b1508
- Downward API support for status.hostIPs https://github.com/kubernetes/kubernetes/pull/101566/commits/3281e7e72b39119de54773cb679d096e7b34fd1b